### PR TITLE
Feature/gp2 1180 travel and business policies page

### PR DESCRIPTION
**Pull request by jakehopking**
(Please add a sentence or two explaining the context of this PR. Reviewers can go to the ticket for detail.)
CONTEXT: This changeset adds

- New Travel Plan section to Export plan.
- Add new generic API endpoint.
- Add generic Learning component (independent of FormElements).

_Tick or delete as appropriate:_

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1180
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [X] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

screencapture-greatcms-trade-great-8020-export-plan-section-travel-and-business-policies-2021-02-17-16_23_24

### CODE REVIEW

File: core/sass/components/_forms.scss [hunk 1]
```diff
@@ -99,3 +99,62 @@
...
   }
 }
+
+.form-group--small {
+  label {
+    font-size: var(--font-size-body-3);
+  }
+}
+
+.great-radio {
```
Comment (dahfool) [inline core/sass/components/_forms.scss:109]: @jakehopking Should these styles be part of great-styles.

Comment (jakehopking) [reply core/sass/components/_forms.scss:109]: I think we'll have a clearer view on this when we have our conversation with Michal next week.

File: exportplan/templates/exportplan/sections/travel-and-business-policies.html [hunk 2]
```diff
@@ -1,20 +1,189 @@
...
+    <div class="c-1-12-m c-1-4-xl">&nbsp;</div>
+  </div>
+</section>
+
 {% endblock %}
+
+{% block body_js %}
+{{ block.super }}
+<script type="text/javascript">
+  magna.travelPlanSnapshot({
```
Comment (bobmeredith) [inline exportplan/templates/exportplan/sections/travel-and-business-policies.html:98]: It feels very wrong to have all this JSON in an HTML template.
Suggest you split it up - perhaps put each json chunk in its own file and {% include %} it into this script block - or perhaps {% include %} the whole script block.

**Review by bobmeredith — COMMENTED** [review]
Needs unit tests on the React components.
This is a big PR. If all this stuff was in one ticket, I suggest you push back and get the DM to split into several smaller tickets.

File: react-components/src/components/Learning/Learning.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import React, { useState, memo } from 'react'
```
Comment (dahfool) [inline react-components/src/components/Learning/Learning.jsx:1]: @jakehopking This component should re-use in `FormGroup` avoid duplication.

File: react-components/src/components/TravelPlan/VisaInformation/VisaInformation.jsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            type="radio"
+            value="false"
+            checked={needVisa === false}
+            onChange={(event) => onNeedVisaChange(event)}
+          />
+          <label htmlFor="need-visa-false" className="great-radio__label">
+            I don't need a visa
+          </label>
+        </div>
+        <div className="great-radio">
```
Comment (dahfool) [inline react-components/src/components/TravelPlan/VisaInformation/VisaInformation.jsx:60]: Should be extracted a form element component.

Comment (jakehopking) [reply react-components/src/components/TravelPlan/VisaInformation/VisaInformation.jsx:60]: I think we'll have a clearer view on this when we have our conversation with Michal next week.

File: react-components/src/components/Learning/Learning.jsx (hunk 3 above)
Comment (jakehopking) [reply react-components/src/components/Learning/Learning.jsx:1]: Agreed.

File: exportplan/templates/exportplan/sections/travel-and-business-policies.html (hunk 2 above)
Comment (jakehopking) [reply exportplan/templates/exportplan/sections/travel-and-business-policies.html]: Updated.

File: react-components/src/components/Learning/Learning.jsx (hunk 3 above)
Comment (jakehopking) [reply react-components/src/components/Learning/Learning.jsx:1]: Done.

File: core/context_processors.py [hunk 5]
```diff
@@ -23,6 +23,8 @@ def env_vars(request):
     data = dict()
     data['great_support_email'] = settings.GREAT_SUPPORT_EMAIL
     data['dit_on_govuk'] = settings.DIT_ON_GOVUK
+    data['travel_advice_covid19'] = settings.TRAVEL_ADVICE_COVID19
```
Comment (depsiatwal) [inline core/context_processors.py:26]: doesn't appear this link needs to be in context_processor. 
As it's not required on post pages and isn't generic. 
Please move to BusinessAndTravelView

Comment (jakehopking) [reply core/context_processors.py:26]: Updated.

**Review by depsiatwal — APPROVED** [review]
happy with the changes I proposed :-)

**Pull request merged**